grpc-client-utils/src/main/java/org/hypertrace/core/grpcutils/client/GrpcChannelRegistry.java

```diff
@@ -12,19 +12,39 @@
   private final Map<String, ManagedChannel> channelMap = new ConcurrentHashMap<>();
   private volatile boolean isShutdown = false;
 
+  /** Use either {@link #forSecureAddress(String, int) or} */
```

Comment:
```suggestion
  /** Use either {@link #forSecureAddress(String, int)} or {@link #forPlaintextAddress(String, int)}  */
```

Reply:
whoops, did I not finish writing that...

Reply:
fixed